readme.md

```diff
@@ -194,9 +194,9 @@
 - [Backbone](https://github.com/sadcitizen/awesome-backbone#readme) - App framework.
 - [HTML5](https://github.com/diegocard/awesome-html5#readme) - Markup language used for websites & web apps.
 - [SVG](https://github.com/willianjusten/awesome-svg#readme) - XML-based vector image format.
-- [Canvas](https://github.com/raphamorim/awesome-canvas#readme)
-- [KnockoutJS](https://github.com/dnbard/awesome-knockout#readme)
-- [Dojo Toolkit](https://github.com/petk/awesome-dojo#readme)
+- [Canvas](https://github.com/raphamorim/awesome-canvas#readme) - HTML5 and javascript charting library.
```

Comment:
Should this be "HTML5 and JavaScript charting library."?

Reply:
Yes

---

Just a tip, it's better to just do the change instead of asking first. Now we need yet another back and forth.

Reply:
Sure, thanks for the help!